Bottom: ba6b8c94244b8c6ce426b33fa7f799ad9cf0ea8d
Top:    d91d394dc7ea9418cc95defa7e1591088d23fd89
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-02 16:02:27 +0800

Refresh of vhost-part

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index c0cbc21..cfda146 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -556,8 +556,6 @@ static netdev_tx_t macvlan_start_xmit(struct sk_buff *skb,
 	unsigned int len = skb->len;
 	int ret;
 
-	printk("start xmit!\n");
-
 	if (unlikely(netpoll_tx_running(dev)))
 		return macvlan_netpoll_send_skb(vlan, skb);
